Modules/Quotas/test/Quotas.Infrastructure.Tests/Tests/Repositories/RelationshipsRepositoryTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Infrastructure.Persistence.Database;
+using Backbone.Modules.Quotas.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Relationships.Domain.Aggregates.Relationships;
+using Backbone.Modules.Relationships.Infrastructure.Persistence.Database;
+using Backbone.Tooling;
+using Backbone.UnitTestTools.TestDoubles.Fakes;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+
+using static Backbone.Modules.Relationships.Domain.Tests.TestHelpers.TestData;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Tests.Tests.Repositories;
+public class RelationshipsRepositoryTests
+{
+    private readonly RelationshipsDbContext _relationshipsArrangeContext;
+    private readonly QuotasDbContext _actContext;
+
+    public RelationshipsRepositoryTests()
+    {
+        AssertionScope.Current.FormattingOptions.MaxLines = 1000;
+
+        var connection = FakeDbContextFactory.CreateDbConnection();
+        (_relationshipsArrangeContext, _, _) = FakeDbContextFactory.CreateDbContexts<RelationshipsDbContext>(connection);
+        (_, _, _actContext) = FakeDbContextFactory.CreateDbContexts<QuotasDbContext>(connection);
+
+        SystemTime.Set(DateTime.UtcNow);
```

Comment:
Why?

Reply:
I thought it ads value to the tests, but you are right it is unnecessary. Changes made.